raiden/ui/console.py

```diff
@@ -235,7 +235,7 @@ def _run(self):  # pylint: disable=method-hidden
         print_usage()
 
         # Remove handlers that log to stderr
-        root = structlog.get_logger()
+        root = logging.get_logger()
         for handler in root.handlers[:]:
```

Comment:
I'll check these; it's possible it can be removed as the logging is configured elsewhere.

Reply:
Feel free to take over this PR, just wanted to get this started as I want to test the RPCClient PR.